### PR TITLE
Add a start over option at adipose

### PR DESCRIPTION
**Pull request by gh0staria**

### CODE REVIEW

**Comment by gh0staria** [comment]
Please review this, @casey-collab !

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: english/adipose/adipose.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 You remember you took your diet pills before going to sleep.
 The marshmallows are in fact your Adipose!
 You die from fat drain.
+
+Click [here](/english/marshmallow.md) to start over.
```
Comment (casey-collab) [inline english/adipose/adipose.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by gh0staria** [comment]
Alright, I fixed the link! @casey-collab

File: english/adipose/adipose.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 You remember you took your diet pills before going to sleep.
 The marshmallows are in fact your Adipose!
 You die from fat drain.
+
+Click [here](/english/marshmallow.md/) to start over.
```
Comment (casey-collab) [inline english/adipose/adipose.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**